.github/workflows/go.yml

```diff
@@ -78,6 +78,10 @@ jobs:
           path: source
           fetch-depth: 0
 
+      - name: Start pulling environment in the background
+        run: make env-pull &
```

Comment:
Looks like there missing something after `&`

Reply:
No, that sends `make env-pull` command to bash's background to run it and the next step in parallel to make CI a bit faster